### PR TITLE
Cards with rating and new button // Refactoring code // CSS

### PR DESCRIPTION
**Pull request by felixcanditt**
Thanks for your review :)

I changed the following:

- added rating to library cards
- replaced checkbox by button in watchlist cards
- refactoring code
- CSS: styled forms and cards, changed colors

### CODE REVIEW

**Review by sacidcelik — APPROVED** [review]
good job! only some remarks to consider if you want to :)

File: client/src/components/LibraryCard.js [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+      {displayCategory(item.category)}
+
+      {item.rating ? (
+        <Stars>
+          {displayStar(1)}
+          {displayStar(2)}
+          {displayStar(3)}
+          {displayStar(4)}
+          {displayStar(5)}
+        </Stars>
```
[the remark this answers is not in the document]
Comment (felixcanditt) [reply client/src/components/LibraryCard.js:39]: didnt work, ill keep it like that :D

File: client/src/components/WatchlistCard.js [hunk 2]
```diff
@@ -11,38 +12,45 @@ export default function WatchlistCard({
 }) {
   return (
     <Card>
-      <p>{item.title}</p>
+      {item.title ? <h4>{item.title}</h4> : <h4>Ohne Titel</h4>}
```
[the remark this answers is not in the document]
Comment (felixcanditt) [reply client/src/components/WatchlistCard.js:15]: Yes I allow that :)

File: client/src/components/WatchlistCard.js [hunk 3]
```diff
@@ -11,38 +12,45 @@ export default function WatchlistCard({
...
       <WatchlistCardDetails item={item} />
-      <WatchlistCheckbox item={item} onCheckItem={onCheckItem} />
-      <button onClick={() => onSetItemToBeEdited(item)}>bearbeiten</button>
-      <button onClick={() => onRemoveFromWatchlist(item)}>löschen</button>
+      <Buttons>
+        <button onClick={() => onCheckItem(item)}>
+          {displayButtonText(item.category)}
```
[the remark this answers is not in the document]
Comment (felixcanditt) [reply client/src/components/WatchlistCard.js:19]: thanks for the hint, noted for later

**Pull request merged**